cmd/solana_exporter/slots.go

```diff
@@ -245,3 +267,28 @@ func getEpochBounds(info *rpc.EpochInfo) (int64, int64) {
 	firstSlot := info.AbsoluteSlot - info.SlotIndex
 	return firstSlot, firstSlot + info.SlotsInEpoch - 1
 }
+
+// fetchAndEmitInflationRewards fetches and emits the inflation rewards for the configured inflationRewardAddresses
+// at the provided epoch
+func (c *SlotWatcher) fetchAndEmitInflationRewards(ctx context.Context, epoch int64) error {
+	epochStr := fmt.Sprintf("%d", epoch)
+	klog.Infof("Fetching inflation reward for epoch %v ...", epochStr)
+
+	ctx, cancel := context.WithTimeout(ctx, httpTimeout)
+	defer cancel()
+
+	rewardInfos, err := c.client.GetInflationReward(
+		ctx, c.inflationRewardAddresses, rpc.CommitmentFinalized, &epoch, nil,
+	)
+	if err != nil {
+		return err
```

Comment:
```suggestion
		return fmt.Errorf("error fetching inflation rewards: %w", err)
```

Might be a bit more friendly to the operator. If the rpc fails, it is going to be a relatively generic error that won't be super obvious.